user-guide/Advanced_Modules/SRM/The_Services_module/SRM_Services_definitions.md

```diff
@@ -72,6 +72,16 @@ To do so:
 
 1. You can then further refine the created setup if necessary:
 
+   - To further refine parameters for a particular node, select the node, and, in the lower right corner, specify a parameter label and/or select one of the available profiles for the parameter. From DataMiner 10.0.9 onwards, a toggle button is also available next to the *Profile* box, which determines if the *By reference* option is selected by default during booking creation. If *By reference* is selected, users will not be able to adapt the value for the selected profile instance during booking creation.
```

Comment:
...specify a label and optionally select a default profile instance or provide parameter values...

The SRM framework requires a label on each node

Profile Instance here is the default profile instance used at booking creation. User can then select a different profile instance when creating the booking.